src/platforms/php/guides/laravel/other-versions/lumen.mdx

```diff
@@ -62,25 +57,32 @@ SENTRY_LARAVEL_DSN=___PUBLIC_DSN___
 
 ### Verify with Artisan
 
-You can test your configuration using the provided `artisan` command:
+You can test your configuration using the provided `sentry:test` artisan command:
 
 ```shell
-php artisan sentry:test --transaction
-[Sentry] DSN discovered!
-[Sentry] Generating test Event
-[Sentry] Sending test Event
-[Sentry] Transaction sent: b22f130764c940c69d6860c212849d48
-[Sentry] Event sent with ID: 4cde238791a9433c8876f4a545d9a95b
+php artisan sentry:test
 ```
 
-## Verify with code
 
-You can easily verify that Sentry is capturing errors in your Lumen application by creating a debug route that will throw an exception:
+### Verify with code
+
+You can verify that Sentry is capturing errors in your Laravel application by creating a route that will throw an exception:
 
-```php
+```php {filename:routes/web.php}
 Route::get('/debug-sentry', function () {
     throw new Exception('My first Sentry error!');
 });
 ```
 
 Visiting this route will trigger an exception that will be captured by Sentry.
+
+
+## Local development and testing
+
+When Sentry is installed in your application it will also be active when you are developing or running tests.
+
+You most likely don't want errors to be sent to Sentry when you are developing or running tests, set the DSN value to `null` to disable sending errors to Sentry.
```

Comment:
```suggestion
You most likely don't want errors to be sent to Sentry when you are developing or running tests. To avoid this, set the DSN value to `null` to disable sending errors to Sentry.
```